packages/runner-cypress/e2e/en-keyboard-navigation.feature

```diff
@@ -0,0 +1,101 @@
+Feature: Keyboard Navigation
+
+  Scenario: Focus on app logo
+    Given I visit path "https://e2e-test-quest.github.io/weather-app/"
+    When I start a keyboard navigation from the top of the page
+    Then the element with role "link" and name "Weather App's Logo" should be keyboard focused
+    And I should see a link named "Weather App's Logo" keyboard focused
+
+  Scenario: Focus on app link
+    Given I visit path "https://e2e-test-quest.github.io/weather-app/"
+    And I start a keyboard navigation from the top of the page
+    When I go to next keyboard element
+    Then the element with role "link" and name "Home" should be keyboard focused
```

Comment:
Tu as fait exprès d'utiliser la phrase tech pour avoir un test dessus ? 
Je m'attendais à avoir the link named "Home" should be keyboard focused

Reply:
En fait je n'étais pas allé au bout de ce que j'avais prévu.
L'idée c'est que pour les 3 premier scénarios (démarrage de la navigation, next et previous), j'appelle lors de la vérification à la fois la phrase tech et la phrase normale pour m'assurer que les 2 sont ok.
Pour les autres scénarios ensuite je n'utilise plus les phrases tech.
J'ai fait la correction, et j'ai également passé le fichier fr en francçais, car ce n'était pas le cas